react/Field/index.jsx

```diff
@@ -78,6 +79,7 @@ const Field = props => {
       case 'textarea':
         return (
           <Textarea
+            disabled={disabled}
```

Comment:
Does this component can handle `disabled` as well?

Reply:
It doesn't.

I suggest to remove this line.

Reply:
Spoiler: I've just removed it.

Reply:
It would have been nice to add this behavior to Textarea component instead of removing the line. 😜
For consistency ✊

Reply:
I agree, it would have been nice. 😏

Reply:
Done.

![capture d ecran 2019-01-18 a 12 55 43](https://user-images.githubusercontent.com/776764/51385752-75c10c00-1b20-11e9-8bef-20c34dafafd5.png)

Reply:
You're the man. 👍